### PR TITLE
Feature 2452 update release guide

### LINKED ISSUE #2452: Documentation: Update Release Guide for MET releases to update version numbers in the installation.rst in the MET User's Guide
## Describe the Task ##
Update Release Guide for MET releases to update version numbers in the installation.rst in the MET User's Guide.   See this [comment](https://github.com/dtcenter/MET/issues/2588#issuecomment-1856417264) in MET issue 2588 for further guidance.

### Time Estimate ###
1 to 3 days of work

### Sub-Issues ###
Consider breaking the task down into sub-issues.
- [ ] *Add a checkbox for each sub-issue here.*

### Relevant Deadlines ###
6.0.0 release

### Funding Source ###
?

### Assignee ###
- [x] Select **engineer(s)** or **no engineer** required
- [ ] Select **scientist(s)** or **no scientist** required

### Labels ###
- [x] Select **component(s)**
- [x] Select **priority**
- [x] Select **requestor(s)**

### Projects and Milestone ###
- [x] Select **Repository** and/or **Organization** level **Project(s)** or add **alert: NEED CYCLE ASSIGNMENT** label
- [x] Select **Milestone** as the next official version or **Future Versions**

## Define Related Issue(s) ##
Consider the impact to the other METplus components.
- [x] [METplus](https://github.com/dtcenter/METplus/issues/new/choose), [MET](https://github.com/dtcenter/MET/issues/new/choose), [METdataio](https://github.com/dtcenter/METdataio/issues/new/choose), [METviewer](https://github.com/dtcenter/METviewer/issues/new/choose), [METexpress](https://github.com/dtcenter/METexpress/issues/new/choose), [METcalcpy](https://github.com/dtcenter/METcalcpy/issues/new/choose), [METplotpy](https://github.com/dtcenter/METplotpy/issues/new/choose)

## Task Checklist ##
See the [METplus Workflow](https://metplus.readthedocs.io/en/latest/Contributors_Guide/github_workflow.html) for details.
- [ ] Complete the issue definition above, including the **Time Estimate** and **Funding Source**.
- [ ] Fork this repository or create a branch of **develop**.
Branch name: `feature_<Issue Number>_<Description>`
- [ ] Complete the development and test your changes.
- [ ] Add/update log messages for easier debugging.
- [ ] Add/update unit tests.
- [ ] Add/update documentation.
- [ ] Add any new Python packages to the [METplus Components Python Requirements](https://metplus.readthedocs.io/en/develop/Users_Guide/overview.html#metplus-components-python-requirements) table.
- [ ] Push local changes to GitHub.
- [ ] Submit a pull request to merge into **develop**.
Pull request: `feature <Issue Number> <Description>`
- [ ] Define the pull request metadata, as permissions allow.
Select: **Reviewer(s)** and **Development** issues
Select: **Repository** level development cycle **Project** for the next official release
Select: **Milestone** as the next official version
- [ ] Iterate until the reviewer(s) accept and merge your changes.
- [ ] Delete your fork or branch.
- [ ] Close this issue.

### PR DESCRIPTION
**Pull request by jprestop**
## Pull Request Testing ##

- [x] Describe testing already performed for these changes:</br>
Reviewed the two modified sections in RTD:

- [Update Version Number](https://metplus.readthedocs.io/en/feature_2452_update_release_guide/Release_Guide/met_official.html#update-version-number)
- [Update tar_files Link](https://metplus.readthedocs.io/en/feature_2452_update_release_guide/Release_Guide/met_official.html#update-tar-files-link)

- [x] Recommend testing for the reviewer(s) to perform, including the location of input datasets, and any additional instructions:</br>
Review the two modified sections, make any suggestions for changes, and ensure that all tests pass:
- [Update Version Number](https://metplus.readthedocs.io/en/feature_2452_update_release_guide/Release_Guide/met_official.html#update-version-number)
- [Update tar_files Link](https://metplus.readthedocs.io/en/feature_2452_update_release_guide/Release_Guide/met_official.html#update-tar-files-link)

- [x] Do these changes include sufficient documentation updates, ensuring that no errors or warnings exist in the build of the documentation? **[Yes]**

- [x] Do these changes include sufficient testing updates? **[Yes]**

- [x] Will this PR result in changes to the test suite? **[No]**</br>
If **yes**, describe the new output and/or changes to the existing output:</br>

- [x] Do these changes introduce new SonarQube findings? **[No]**</br>
If **yes**, please describe:

- [x] Please complete this pull request review by **[20240920]**.</br>

## Pull Request Checklist ##
See the [METplus Workflow](https://metplus.readthedocs.io/en/latest/Contributors_Guide/github_workflow.html) for details.
- [x] Add any new Python packages to the [METplus Components Python Requirements](https://metplus.readthedocs.io/en/develop/Users_Guide/appendixA.html#metplus-components-python-packages) table.
- [x] For any new datasets, an entry to the [METplus Verification Datasets Guide](https://metplus.readthedocs.io/en/latest/Verification_Datasets/index.html).
- [x] Review the source issue metadata (required labels, projects, and milestone).
- [x] Complete the PR definition above.
- [x] Ensure the PR title matches the feature or bugfix branch name.
- [x] Define the PR metadata, as permissions allow.
Select: **Reviewer(s)** and **Development** issue
Select: **Milestone** as the version that will include these changes
Select: **Coordinated METplus-X.Y Support** project for bugfix releases or **METplus-Wrappers-X.Y.Z Development** project for official releases
- [x] After submitting the PR, select the :gear: icon in the **Development** section of the right hand sidebar. Search for the issue that this PR will close and select it, if it is not already selected.
- [ ] After the PR is approved, merge your changes. If permissions do not allow this, request that the reviewer do the merge.
- [ ] Close the linked issue and delete your feature or bugfix branch from GitHub.

### CODE REVIEW

File: docs/Release_Guide/release_steps/met/update_tar_files_link_official.rst [hunk 1]
```diff
@@ -0,0 +1,8 @@
+Update tar_files Link
+^^^^^^^^^^^^^^^^^^^^^
+
+On mohawk, in /d2/www/dtcenter/dfiles/code/METplus/MET/installation/, update
+the link for tar_files.latest.tgz, to point to the latest version of the
+tar_files.met-base-vX.Y.tgz file.
```
Comment (JohnHalleyGotway) [inline docs/Release_Guide/release_steps/met/update_tar_files_link_official.rst]:
```suggestion
tar_files.met-base-vX.Y.tgz file (viewable at https://dtcenter.ucar.edu/dfiles/code/METplus/MET/installation/).
```

File: docs/Release_Guide/release_steps/met/update_tar_files_link_official.rst [hunk 2]
```diff
@@ -0,0 +1,8 @@
+Update tar_files Link
+^^^^^^^^^^^^^^^^^^^^^
+
+On mohawk, in /d2/www/dtcenter/dfiles/code/METplus/MET/installation/, update
```
Comment (JohnHalleyGotway) [inline docs/Release_Guide/release_steps/met/update_tar_files_link_official.rst]:
```suggestion
On the DTC web server machine (i.e. mohawk), in /d2/www/dtcenter/dfiles/code/METplus/MET/installation/, update
```
mohawk will eventually be replaced and renamed.

**Review by JohnHalleyGotway — DISMISSED** [review]
@jprestop thanks for making these changes. I'll mark this as approved but do recommend that you consider the 2 minor tweaks I suggested.

File: docs/Release_Guide/release_steps/met/update_tar_files_link_official.rst (hunk 1 above)
Comment (jprestop) [reply docs/Release_Guide/release_steps/met/update_tar_files_link_official.rst]: @JohnHalleyGotway when you say "and confirm the result at https://dtcenter.ucar.edu/dfiles/code/METplus/MET/installation/."  Do you mean by checking the size of the file?

File: docs/Release_Guide/release_steps/met/update_tar_files_link_official.rst (hunk 2 above)
Comment (jprestop) [reply docs/Release_Guide/release_steps/met/update_tar_files_link_official.rst]: @JohnHalleyGotway Thank you for making this suggested change.  I'lll update this now.

**Pull request merged**